roles/splunk/defaults/main.yml

```diff
@@ -75,3 +75,16 @@ add_pstack_script: false # Set to true to install a pstack generation script for
 configure_dmesg: false
 install_utilities: false # Set to true to install the list of packages defined in the linux_packages var after installing splunk
 use_tuned_thp: false
+# Firewall configs
+configure_firewall: false # Whether or not to configure the firewall service on your machine, if set to true, opens firewall ports using UFW (default) or Firewalld depending on OS
+splunk_firewall_service: splunk # The name of the Splunk firewall service to install for firewalld
+# Firewall port presets - reference these in group_vars to assign them to splunk
+splunkweb_port: {protocol: "tcp", number: 8000}
+splunkhec_port: {protocol: "tcp", number: 8088}
+splunktcpin_port: {protocol: "tcp", number: 9997}
+splunkapi_port: {protocol: "tcp", number: "{{ splunkd_port }}"}
+splunkidxcrep_port: {protocol: "tcp", number: "{{ splunk_idxc_rep_port }}"}
+splunkshcrep_port: {protocol: "tcp", number: "{{ splunk_shc_rep_port }}"}
+splunk_firewall_ports: # List of ports to allow through local firewall in dict form
```

Comment:
Do we really need to define this? This will open the ports on UFs, and since the UF is no longer listening on 0.0.0.0:8089 by default, I don't think we need that.

Reply:
Thankfully it will not - in [`configure_os.yml`](https://github.com/splunk/ansible-role-for-splunk/pull/157/commits/f93c8a18104136f20458ce214e3770068b9d1f80#diff-c80bf73fffa3043683ec15ad3dc6d7b4e303e31a2aacd745625d7e1b09218f96) I have this so that we don't install the firewall unless it's in the `full` category:

```yaml
- name: Configure firewall service
  include_tasks: "configure_firewall.yml"
  when:
    - firewall_service != 'undefined'
    - configure_firewall != false
    - "'full' in group_names"
```